### DISCUSSION TITLE
V1リリースに必要な画像を用意する！

### DISCUSSION THREAD

**Discussion by kazzyfrog** [lgtm-factory/lgtm-factory#155] [Scrap]
メモしながら進めていきますー！
- at: #145 

---

- Github ソーシャルプレビュー・READMEのヘッダー画像
- OG画像
- ファビコン類（アップルアイコン、など）

その他：
- エラー画像： #92
- 404ページ：

**Comment by kazzyfrog** [comment]
Github ソーシャルプレビュー
> 画像は少なくとも640×320px（最適な表示のために1280×640px）である必要があります。

**Reply by kazzyfrog** [reply]
![Group 167](https://github.com/user-attachments/assets/0faa85c2-c58f-46dd-911d-0a4e10ad5319)

**Comment by kazzyfrog** [comment]
OG、Twitter：
![Group 168](https://github.com/user-attachments/assets/cd7a78cb-0c30-4c1e-803e-f25e94af04b3)

**Comment by kazzyfrog** [comment]
ファビコン、アイコン
![Group 154](https://github.com/user-attachments/assets/b8ba200c-1343-465c-825c-15a2ca245f21)

**Comment by kagomen** [comment]
作成ありがとうございます！
良いと思いますー！！！👍🌈
